src/transformers/configuration_bart.py

```diff
@@ -28,6 +28,7 @@
     "facebook/bart-large-cnn": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/bart-large-cnn/config.json",
     "facebook/bart-large-xsum": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/bart-large-xsum/config.json",
     "facebook/mbart-large-en-ro": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/mbart-large-en-ro/config.json",
+    "yjernite/bart_eli5": "https://s3.amazonaws.com/models.huggingface.co/bert/yjernite/bart_eli5/config.json",
```

Comment:
Apparently this is no longer needed, right @julien-c ?

Reply:
It's not used anymore but we still expect a dict at the moment. 

The case is slightly different between model weights (it's now a list) and configs.